README.md

```diff
@@ -37,8 +37,8 @@ npm install
 pip install -r requirements.txt
 ```
   
-We require all contributors to write docsting for other contributors easy to follow.  
-One of the software development process is TDD(Test-driven Development). We highly recommended you *write at least one test before code*  
+We require all contributors to write docstring for other contributors easy to follow.  
```

Comment:
May want to rephrase to
`We require all contributors to write docstring for other contributors so the codes are easy to follow.  `

Reply:
resolved.